src/tools/github2gitlab.rst

```diff
@@ -0,0 +1,73 @@
+=====================================
+Transition your Github Repo to Gitlab
+=====================================
+
+Create Gitlab Account
+=====================
+
+..
+  gitlab.com > create account
+
+Project Members
```

Comment:
I'd switch the ordering of these two sections - otherwise people may try to change their remote before the project's been switched over!